spec/polaris-management-service.yml

```diff
@@ -911,6 +911,7 @@ components:
             - OAUTH
             - BEARER
             - SIGV4
+            - NO_AUTH
```

Comment:
Beside that this is a breaking change, any user facing API change requires a discussion on the dev mailing list.

Reply:
+1 to dev list discussion

Reply:
It is an `enum` in older clients - and Java enum `valueOf` throws and does not gracefully handle unknown values. Hence it is a breaking change.

Reply:
> Separately, if you think it's a good idea to harden our Java client against this kind of enum change, I think that would be a great contribution.

I guess that it would be a hard fight to remove the code generator, mind proposing that change to get that implemented?

Reply:
In fact, is the Java client already set up to treat this as NULL_TYPE?

```java
if (authTypeCode < 0 || authTypeCode >= REVERSE_MAPPING_ARRAY.length) {
      return NULL_TYPE;
    }
```

Reply:
And the other way around?

Reply:
Sorry, what do you mean by the other way around? Does this relate to the doc linked above?